gui/src/components/Layout.svelte

```diff
@@ -1,27 +1,45 @@
 <script lang="ts">
-  import * as router from 'svelte-spa-router';
   import VersionInfo from './VersionInfo.svelte';
   import NavbarButton from './nav/NavbarButton.svelte';
-  import FeedbackSVG from './mono/feedback.svelte';
+  import FeedbackSVG from './mono/FeedbackSVG.svelte';
+  import PreferencesSVG from './mono/PreferencesSVG.svelte';
+  import { guiTheme, setGuiTheme } from '../lib/gui-theme';
+  import { onMount } from 'svelte';
+  import * as router from 'svelte-spa-router';
 
-  const goHome = () => {
-    router.push('#/');
-  };
+  onMount(() => {
```

Comment:
I discuss `setGuiTheme` elsewhere, so go see that comment.

This is where you'd want something like:

```
$: {
  doWhateverTo(document.body.classList, $guiTheme);
}
```